community/approved-plugins.json

```diff
@@ -16,6 +16,11 @@
   },
   {
     "name": "@nxtend/capacitor",
+    "description": "An Nx plugin for developing applications using Firebase",
```

Comment:
Do you want to change this description with the one below 😅

Reply:
Dang dude. I mess these up more than my actual code.